assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -17,6 +17,7 @@ angular.module('openshiftConsole')
       $location.url(redirect);
     }
 
+    // FIXME: This is v1beta1, but it's what's in the ruby-helloworld-sample template
```

Comment:
Not sure what to do here. It's working, but the next few lines are clearly v1beta1.